client/src/App.js

```diff
@@ -274,6 +273,33 @@ class App extends Component {
     });
   }
 
+  async createDownload(query) {
+    
+    this.setState({ isLoading: true });
+    let data = null;
+    try {
+      data = await exportService.save(query);
+    } catch (exc) {
+      exc.customMessage = "Error in exporting graph";
+      eventService.publishError(exc);
+    }
+
+    if (data) {
+      const blob = new Blob([ JSON.stringify(data) ], { type: "application/json" });
+      const downloadUrl = URL.createObjectURL(blob);
+      let downloadLink = document.createElement("a");
+      downloadLink.setAttribute("href", downloadUrl);
+      downloadLink.setAttribute("download", "Datamodel");
+      downloadLink.style.visibility = 'hidden';
+      document.body.appendChild(downloadLink);
+      downloadLink.click();
+      document.body.removeChild(downloadLink);
+      setTimeout(() => URL.revokeObjectURL(downloadUrl), 100);
```

Comment:
Why 100ms? What are we waiting for here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL/revokeObjectURL As far as I could see in the existing code (ExportComponent) and some examples in the Internet its to let the browser know not to keep the reference to the file any longer.